meb/utils/utils.py

```diff
@@ -101,8 +101,10 @@ class Printer:
     def __init__(self, config, split_column: str):
         self.cf = config
         self.split_column = split_column
-        if self.cf.action_units:
+        if 'AU' in self.cf.action_units[0]:
             self.label_type = "au"
+        elif 'emotion' in self.cf.action_units[0]:
```

Comment:
Why is 'emotion' in action units? Need a better way to separate the two

Reply:
i guess self.cf.action_units can be renamed to training_labels or something related.